scripts.py

```diff
@@ -1221,8 +1220,16 @@ def do_run(self):
 
 
 
-class CheckContributorNames(IdentifierInputScript):
-    """ TODO """
+class CheckContributorNamesInDB(IdentifierInputScript):
+    """ Checks that contributor sort_names are display_names in 
+    "last name, comma, other names" format.  
+
+    NOTE:  There's also CheckContributorNamesOnWeb in metadata, 
+    it's a child of this script.  Use it to check our knowledge against 
+    viaf, with the newer better sort_name selection and formatting.
+
+    TODO: make sure don't start at beginning again when interrupt while batch job is running.
```

Comment:
An `IdentifierSweepMonitor` can do this, but right now you can't combine an `IdentifierInputScript` with a `RunMonitorScript`.

Is it worth making the change? I'm not sure how often we plan on running this script.